thoth/storages/graph/postgres.py

```diff
@@ -3982,7 +3982,7 @@ def sync_analysis_result(self, document: dict) -> None:
                 sw_class = SoftwareEnvironment
                 python_version = image_name.rsplit("-", maxsplit=1)[1]   # pyXX
                 python_version = python_version[2:3] + "." + python_version[3:]   # take first digit and put . after it
-                if not re.match(r"^\d\.\d*$", python_version):
+                if not re.match(r"^\d(\.\d+)*$", python_version):
```

Comment:
We expect both to be present: `r"^\d\.\d+$"`

Reply:
Actually, this will work because the dot will be always present (based on the code above) forcing the minor version to be checked. IMHO worth to be explicit here

Reply:
I'm unsure what you mean.  Should we make sure there is a minor version attached or not?

Reply:
> I'm unsure what you mean. Should we make sure there is a minor version attached or not?

Yes, the minor version should always present. The major version as well.